packages/main/src/themes/Input.css

```diff
@@ -221,12 +221,12 @@
 	background: var(--sapField_Hover_BackgroundStyle);
 	background-color: var(--sapField_Hover_Background);
 }
-:host(:not([value-state]):not([readonly])[focused]:hover),
-:host([value-state="None"]:not([readonly])[focused]:hover) {
+:host(:not([value-state]):not([opened]):not([readonly])[focused]:hover),
+:host([value-state="None"]:not([opened]):not([readonly])[focused]:hover) {
```

Comment:
When, if the ui5-select is open and you hover its input field, the hover styles are not applied